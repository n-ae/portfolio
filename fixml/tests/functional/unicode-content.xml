<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <Title>测试项目 🚀 émojis</Title>
    <Authors>José María</Authors>
  </PropertyGroup>
</Project>