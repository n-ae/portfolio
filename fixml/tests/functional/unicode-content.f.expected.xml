<?xml version="1.0" encoding="utf-8"?>
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <Title>测试项目 🚀 émojis</Title>
    <Authors>José María</Authors>
  </PropertyGroup>
</Project>
